using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,174 @@
+# Signal
+
+The Signal extension is bundled with the [MobileCore (Android)/ACPCore (iOS)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/) extension and allows you to send postbacks to third-party endpoints and open URLs, such as web URLs or application deep links, when using rules actions in Adobe Experience Platform Launch. 
+
+To send PII data to external destinations, the `PII` action can trigger the Rules engine when certain triggers and traits match. When setting a rule, you can also set the `PII` action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the PII data to a remote server.
+
+To get started with Signal extension, complete the following steps:
+
+1. Add the **Signal** extension to your app.
+2. Define the necessary rules in Experience Platform Launch. 
+3. (Optional) When using Send PII actions in Experience Platform Launch, implement the APIs to Experience Platform Launchcollect PII data and send it to the configured third-party destination.
+
+For more information about creating and configuring a rule in Experience Platform Launch, see [Rules](https://docs.adobe.com/content/help/en/launch/using/reference/manage-resources/rules.html).
+
+## Add the Signal extension to your app
+
+{% tabs %}
+{% tab title="Android" %}
+
+Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using the app's Gradle file.
+
+```java
+implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+```
+
+#### Java
+
+Import the Signal extension in your application's main activity.
+
+```java
+import com.adobe.marketing.mobile.*;
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+​ Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
+
+Add following pods in your `Podfile`:
+
+```shell
+pod 'ACPCore'
+```
+
+Import the Signal libraries:
+
+#### Objective-C
+
+```objective-c
+#import "ACPCore.h"
+#import "ACPSignal.h"
+```
+
+#### Swift
+
+```swift
+import ACPCore
+import ACPSignal
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+#### JavaScript
+
+Install Signal
+
+```jsx
+npm install @adobe/react-native-acpcore
+react-native link @adobe/react-native-acpcore
+```
+
+Importing the Signal extension
+
+```jsx
+import {ACPSignal} from '@adobe/react-native-acpcore';
+```
+
+Note: if using Cocoapods, run:
+
+```
+pod install
+```
+
+{% endtab %}
+{% endtabs %}
+
+### Register Signal extension
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+To call the set up methods that call the [setApplication\(\)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#setapplication) method in the `onCreate()` method:
```

Comment:
there are too many "methods" in this sentence! Please change the end of the sentence as follows:
"...method in `onCreate()`."